pytket/phir/api.py

```diff
@@ -26,10 +27,7 @@
 logger = logging.getLogger(__name__)
 
 
-def pytket_to_phir(
-    circuit: Circuit,
-    qtm_machine: QtmMachine | None = None,
-) -> str:
+def pytket_to_phir(circuit: Circuit, qtm_machine: QtmMachine | None = None) -> str:
```

Comment:
You can ignore this. But I assume QtmMachine stands for Quantinuum Machine. I imagine we might be interested one day of modeling abstract or other machines so it might not be strictly "Quantinuum" machines... but we can deal with that in the future...